src/App.js

```diff
@@ -13,22 +13,55 @@ import DictionaryDisplay from './components/dashboard/container/DictionariesDisp
 import SpecificConcept from './components/dashboard/container/SpecificConcept';
 import DictionaryConcepts from './components/dictionaryConcepts/containers/DictionaryConcepts';
 
+export const NotFound = () => (
+  <div className="not-found">
+    <div className="not-found-container">
+      <h2 className="error-template">Oops!, Page Not Found</h2>
+
+      <div className="error-actions">
+        <a href="/" className="btn btn-primary btn-lg btn-not-found">
+          <span className="fa fa-home" />
+            Take me back home{' '}
+        </a>
+      </div>
+
+    </div>
+  </div>
+);
+
 const App = () => (
   <Provider store={store}>
     <BrowserRouter>
       <div className="App">
         <Navbar />
         <Switch>
           <Route exact path="/" component={Authenticate(Login)} />
-          <Route exact path="/dashboard" component={Authenticate(SourceSearch)} />
-          <Route exact path="/dashboard/concepts" component={Authenticate(ConceptSearch)} />
-          <Route exact path="/dashboard/dictionaries" component={Authenticate(DictionaryDisplay)} />
-          <Route exact path="/dashboard/concepts/:organization/:name" component={Authenticate(SpecificConcept)} />
+          <Route
+            exact
+            path="/dashboard"
+            component={Authenticate(SourceSearch)}
+          />
```

Comment:
please remove the arbitrary spaces to enable better code readability.

Reply:
Thanks though locally its fine with no space I wonder how it comes, let me look into it